Extensions/calc.json

```diff
@@ -0,0 +1,333 @@
+{
+  "author": "add_",
+  "description": "A collection of 7 calculations functions to help find slope, gcf and many other values.\n\nEnjoy :)",
+  "extensionNamespace": "",
+  "fullName": "calculations",
+  "name": "calc",
+  "shortDescription": "A collection of 7 calculations functions to help find slope, gcf and many other values.",
+  "tags": "calculations, math",
+  "version": "1.1.0",
+  "eventsFunctions": [
+    {
+      "description": "Calculates the steepness of a line between two points",
+      "fullName": "Slope",
+      "functionType": "Expression",
+      "name": "slope",
```

Comment:
I think names should be capitalized, not 100% sure though (wait for bouh or 4ian to confirm)

Reply:
No your right they told me the same thing when I submitted the noise extension. So I'll go fix that.